src/main/java/com/aws/greengrass/clientdevices/auth/metrics/VerifyClientDeviceIdentityEvent.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics;
```

Comment:
sounds like this should be in API domain, so `com.aws.greengrass.clientdevices.auth.api.events`?